test/tools/ossfuzz/protoToAbiV2.h

```diff
@@ -418,10 +440,16 @@ class ProtoConverter
 	unsigned m_numStructsAdded;
 	/// Enum stating abiv2 coder to be tested
 	Contract_Test m_test;
+	/// Representation of external parameters
+	std::vector<ParameterPack> m_externalParamsRep;
+	/// Random number generator
+	std::unique_ptr<RandomEngine> m_random;
 	/// Prefixes for declared and parameterized variable names
 	static auto constexpr s_localVarNamePrefix = "lv_";
 	static auto constexpr s_stateVarNamePrefix = "sv_";
 	static auto constexpr s_paramNamePrefix = "p_";
+	/// Maximum number of indirections to test calldata coding
+	static unsigned constexpr s_maxIndirections = 10;
```

Comment:
I guess
```suggestion
	static unsigned constexpr s_maxIndirections = 5;
```
would also be enough :-).

Reply:
Sounds good! Done!